src/configuration.rs

```diff
@@ -18,16 +18,17 @@ pub struct Configuration {
     pub user_agent: &'static str,
     /// Polite crawling delay in milli seconds
     pub delay: u64,
+    /// How many request can be run simultaneously
+    pub concurrency: usize,
 }
 
 impl Configuration {
     pub fn new() -> Self {
         Self {
-            respect_robots_txt: false,
-            verbose: false,
-            blacklist_url: Vec::new(),
-            user_agent: "spider/1.1.2",
-            delay: 0,
+            user_agent: "spider/1.2.1",
```

Comment:
Can you use 

```rs
const VERSION: &'static str = env!("CARGO_PKG_VERSION");
```

See [How can a Rust program access metadata from its Cargo package? - Stack Overflow](https://stackoverflow.com/questions/27840394/how-can-a-rust-program-access-metadata-from-its-cargo-package)

Reply:
It turns out that the simplest tasks are the more difficult. I just don't know how to concatenate two strings.
I don't know what is happening, but string concatenation in Rust is more complicated than thread safety, parralelism and passing closures.

Reply:
Ok so I moved it in issue #11